webapp/packages/plugin-holidays/src/IHoliday.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+export interface IHoliday {
+  startDate: Date;
+  endDate: Date;
+  isHoliday: boolean;
```

Comment:
`holiday.isHoliaday` is a bit weird, it may be active or ongoing or something else.